src/Shared/runtime/ReadMe.SharedCode.md

```diff
@@ -34,3 +34,7 @@ dotnet/aspnetcore code paths:
 ### Running dotnet/aspnetcore tests:
 - `(aspnetcore) PS D:\github\aspnetcore\src\Shared\test\Shared.Tests> dotnet test`
 - `(aspnetcore) PS D:\github\aspnetcore\src\servers\Kestrel\core\test> dotnet test`
+
+## Automation
+
+In dotnet/aspnetcore, the [runtime-sync](https://github.com/dotnet/aspnetcore/actions/workflows/runtime-sync.yml) GitHub action automatically creates PRs to pull in changes from dotnet/runtime.
```

Comment:
```suggestion
In dotnet/aspnetcore, the [runtime-sync](https://github.com/dotnet/aspnetcore/actions/workflows/runtime-sync.yml) GitHub action automatically creates PRs to pull in changes from dotnet/runtime.
In dotnet/runtime, the [aspnet-sync](https://github.com/dotnet/runtime/blob/main/.github/workflows/aspnetcore-sync.yml) GitHub action can be triggered manually to create a PR to pull changes from aspnetcore to runtime.
```

Since this PR modifies the shared folder, please trigger the runtime action when it's merged, otherwise the aspnet action will trigger and try to delete this change. 😂